src/schema/v2/partner/BulkOperation/bulkUpdateArtworksMetadataMutation.ts

```diff
@@ -164,9 +164,22 @@ export const bulkUpdateArtworksMetadataMutation = mutationWithClientMutationId<
     { id, metadata, filters },
     { updatePartnerArtworksMetadataLoader }
   ) => {
-    const gravityOptions = {
-      metadata: metadata,
-      filters: filters,
+    const gravityOptions: any = {}
```

Comment:
Is this `any` here alright? Would it be better to come up with `GravityOptions` interface for defining this? It feels like a special case (in most queries/mutations, we do not have these wrapped objects at this point, wonder what would the ideal here?

Reply:
IMO not necessary -- we're defining the contract via input args via our mutation, which makes things safe on the consumer side (for the most part). To double type it in this way feels kinda like overkill, and we don't typically do it. 

Here's an example that types the inputs for a function that formats the gravity args -- this is mostly for DX: https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/updateArtworkMutation.ts#L187

So you can see how sometimes it makes sense -- especially for inputs that are quite long, and so on. But for a few args here and there, its a use yr best judgement kinda thing 👍